wiki/History_of_osu!/April_Fools/en.md

```diff
@@ -0,0 +1,283 @@
+
+### Free osu! supporter tags
+
+On 1 April 2016 many osu! players were surprised to discover that they had suddenly and inexplicably received an [osu! supporter tag](https://osu.ppy.sh/home/support) despite never having bought or being gifted one. The supporter tag that was given to players was fully functional and acted just like a regular supporter tag; however, the change was rolled back the next day [\[52,53,54,55,56,57,58\]](#references).
+
+### Spinning osu! cookie on website
+
+As part of the handful of April Fools' day jokes for 2016, the [osu! cookie](/wiki/Glossary/Cookie) on the osu! website would occasionally spin 180 degrees clockwise and then quickly spin back upright 180 degrees in the same direction [\[59,60,61,62\]](#references).
+
+---
+
+## 2017
+
+As announced in a [tweet by peppy](https://twitter.com/ppy/status/848021525663842304 "Twitter"), there was no April Fools' day joke for osu! in 2017 due to osu!lazer development.
+
+---
+
+## 2018
+
+### Rotating osu! cookie
+
+On 1 April 2018 the [osu! cookie](/wiki/Glossary/Cookie) on the main menu screen would slowly rotate clockwise as time passed, and the cookie in the song selection screen would slowly rotate anticlockwise. Hovering over these cookies would enlarge them just as usual, but would also cause them to rotate faster [\[63,64,65,66,67,68\]](#references).
+
+---
+
+## 2019
+
+### Sneezing girl sound effect (2019)
+
+For April Fools' day 2019 there was approximately a 1 in 20 chance of hearing the sound effect of a high-pitched girl sneezing upon opening a beatmap [\[69,70,71\]](#references).
+
+---
+
+## 2020
+
+### MillhioreF joins the Featured Artists
+
+[MillhioreF](https://osu.ppy.sh/users/941094)—a long-time osu! moderator, developer and [Easy mod](/wiki/Game_modifier/Easy) player—was announced in [a newspost](https://osu.ppy.sh/home/news/2020-04-01-new-featured-artist-millhioref) to have "joined" the list of [Featured Artists](/wiki/Featured_Artists) as "Millhiore Firianno Biscotti" on 1 April 2020 with a selection of five songs to boot [\[72\]](#references):
+
+- Waltz o' the Irish
+- The Waltzing Irishman
+- An Irish Waltz
+- A Waltz From The Geographical Region Known as Ireland but Also as Éire
+- There's Gold Beneath Your Waltzing Rainbow (feat. Mismagius)
+
+["MillhioreF - Waltz o' the Irish (MillhioreF)"](https://osu.ppy.sh/beatmapsets/73348#osu/326585)—a [long-running joke beatmap](#2009) within the community—was also Loved on 31 March 2020 as part of this joke.
```

Comment:
I don't see any particular relation between this map and the one for the April Fools' 2009, except for both being mapped by MillhioreF. The way you worded it gives the impression that Lemon Tree and Waltz o' the Irish are the same beatmap.
```suggestion
["MillhioreF - Waltz o' the Irish"](https://osu.ppy.sh/beatmapsets/73348#osu/326585) was also [Loved](/wiki/Beatmap/Category#loved) on 31 March 2020 as part of this joke.
```

Reply:
this was just me being stupid; I misread the title